Source/JavaScriptCore/runtime/IntlListFormat.h

```diff
@@ -30,7 +30,6 @@
 
 #if !defined(HAVE_ICU_U_LIST_FORMATTER)
 #if U_ICU_VERSION_MAJOR_NUM >= 67 || (U_ICU_VERSION_MAJOR_NUM >= 66 && USE(APPLE_INTERNAL_SDK))
-#define HAVE_ICU_U_LIST_FORMATTER 1
```

Comment:
I don't think you meant to include this?

Reply:
right